makefile

```diff
@@ -77,6 +77,8 @@ vendor-clean:
 	rm -rf vendor/github.com/mholt/archiver/tarlz4.go
 	rm -rf vendor/github.com/mholt/archiver/tarsz.go
 	rm -rf vendor/github.com/mholt/archiver/tarxz.go
+	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/mongodb/mongo-go-driver/internal/channel_connection.go
+	rm -rf vendor/github.com/mongodb/amboy/vendor/github.com/mongodb/mongo-go-driver/internal/results.go
+	rm -rf vendor/github.com/gogo/protobuf/protoc-gen-gogo/golden_test.go
```

Comment:
seems reasonable to delete the entire packages.  what is it about these specific files?

Reply:
There's nothing wrong with them per se, but they were mostly a minor annoyance because I have a git hook that warns me when files are not `gofmt`'d. I removed these lines from `vendor-clean`.